examples/getting-started/k8s-pod.yaml

```diff
@@ -1,8 +0,0 @@
-apiVersion: v1
-kind: Pod
```

Comment:
Was this accidental?

Reply:
This was the repro case and was accidental to keep in the PR.  Fixed now.